go/README.md

```diff
@@ -1,23 +1,22 @@
 # Tool builder: `gcr.io/cloud-builders/go`
 
-This builder runs the `go` tool (`go build`, `go test`, etc.)
-after placing source in `/workspace` into the `GOPATH` before
-running the tool.
+# Deprecation Notice
 
-# Using `golang` and [Go Modules](https://github.com/golang/go/wiki/Modules)
+This builder is deprecated and will be deleted in an upcoming release. The
+functionality previously provided by this builder is replaced by the use of [Go
+modules](https://github.com/golang/go/wiki/Modules), available since Go 1.11+.
 
-This Builder (`gcr.io/cloud-builders/go`) is not necessary if you're building using
-[Go modules](https://github.com/golang/go/wiki/Modules), available
-in Go 1.11+. You can **build** with the `golang` image (not `gcr.io/cloud-builders/go`) from [Dockerhub](https://hub.docker.com/_/golang) and Google's Container Registry [mirror](https://cloud.google.com/container-registry/docs/using-dockerhub-mirroring):
+In place of this builder, use the [`golang`
+image](https://hub.docker.com/_/golang) from Dockerhub.
 
 ```
 steps:
 # If you already have a go.mod file, you can skip this step.
-- name: mirror.gcr.io/library/golang
+- name: 'mirror.gcr.io/library/golang'
```

Comment:
Are we recommending mirrored images everywhere? It seems kind of confusing, and shouldn't have any effect anyway since we configure the mirror in the daemon.

Reply:
That's a fair point -- I'm going to clean that up in a separate PR...

Reply:
https://github.com/GoogleCloudPlatform/cloud-builders/pull/679